src/Config/SDK/ComponentProvider/OpenTelemetrySdk.php

```diff
@@ -345,6 +482,30 @@ private function getTracerProviderConfig(ComponentProviderRegistry $registry, No
                 ->end()
                 ->append($registry->component('sampler', SamplerInterface::class))
                 ->append($registry->componentList('processors', SpanProcessorInterface::class))
+                ->arrayNode('tracer_configurator/development')
+                    ->addDefaultsIfNotSet()
+                    ->children()
+                        ->arrayNode('default_config')
+                            ->addDefaultsIfNotSet()
+                            ->children()
+                                ->booleanNode('disabled')->defaultFalse()->end()
+                            ->end()
+                        ->end()
+                        ->arrayNode('tracers')
+                            ->arrayPrototype()
+                                ->children()
+                                    ->scalarNode('name')->isRequired()->cannotBeEmpty()->end()
+                                    ->arrayNode('config')
+                                        ->addDefaultsIfNotSet()
+                                        ->children()
+                                            ->booleanNode('disabled')->defaultFalse()->end()
```

Comment:
```suggestion
                                            ->booleanNode('disabled')->isRequired()->end()
```
Same applies to the other `{signal}_configurator/development` nodes.

Reply:
double-checked spec, and updated all of the `disabled` to required